channels/drdynvc/client/drdynvc_main.c

```diff
@@ -242,19 +254,31 @@ static void wts_listener_free(void* arg)
 
 static BOOL channelIdMatch(const void* k1, const void* k2)
 {
+	WINPR_ASSERT(k1);
```

Comment:
These checks are too paranoïd, it's a function that will  be called on every item

Reply:
@hardening that is what `assert` is for ;)
just build with `-DWITH_VERBOSE_WINPR_ASSERT=OFF` and they are gone in your build ;)